docs/dev/news/2907.feat

```diff
@@ -0,0 +1,3 @@
+Added support for Dafny project files. For now the project file only allows specifying which Dafny files to include and exclude, and what options to use.
+The CLI commands that take Dafny files as input, such as build, run, translate, will also accept Dafny project files.
+When using an IDE based on `dafny server`, such as the Dafny VSCode extension, the IDE will look for Dafny project files by traversing up the file tree from the currently opened file. The first found project file will override options specified in the IDE. 
```

Comment:
Will this be immediately true or will the extension need an update first?

Reply:
Immediately true